mappings/net/minecraft/entity/mob/GuardianEntity.mapping

```diff
@@ -40,7 +40,7 @@ CLASS net/minecraft/unmapped/C_kicdatsi net/minecraft/entity/mob/GuardianEntity
 		FIELD f_iwjjscxr owner Lnet/minecraft/unmapped/C_kicdatsi;
 		METHOD <init> (Lnet/minecraft/unmapped/C_kicdatsi;)V
 			ARG 1 owner
-		METHOD test (Ljava/lang/Object;)Z
+		METHOD test test (Ljava/lang/Object;)Z
```

Comment:
There are two tests

Reply:
The obfuscated name is test, I just mapped it with a proper name